apps/wing/src/commands/test.ts

```diff
@@ -153,6 +157,71 @@ async function testSimulator(synthDir: string) {
   }
 }
 
+async function testAwsCdk(synthDir: string): Promise<sdk.cloud.TestResult[]> {
+  const stackName = process.env.CDK_STACK_NAME
+  if (!stackName) {
+    throw new Error(
+      "In order to successfully deploy using AWS-CDK, it is essential to define the environment variable CDK_STACK_NAME."
+    );
+  }
```

Comment:
Is CDK_STACK_NAME needed for the actual deployment, or is it just needed for running `wing compile`?

I think `wing compile -t awscdk` already gives you an error if you do not specify CDK_STACK_NAME, so this check might be redundant.
```suggestion
```